### PR TITLE
New package: kitty-0.10.1

### PR DESCRIPTION
**Pull request by julio641742**
opened after #13507 feedback

### CODE REVIEW

File: srcpkgs/kitty/template [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+conf_files="/etc/kitty/kitty.conf"
+short_desc="Modern, hackable, featureful, OpenGL based terminal emulator"
+maintainer="Julio Galvan <juliogalvan@protonmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/kovidgoyal/kitty"
+distfiles="https://github.com/kovidgoyal/kitty/archive/v${version}.tar.gz"
+checksum=b5e6c07137ebd7977b88f6704341c15b2e43823300435dae9237d320af9a5dec
+
+do_build() {
+	if [ -n "$CROSS_BUILD" ]; then
```
Comment (cr6git) [inline srcpkgs/kitty/template]: It reminds me of one my old pull request: [#10822](https://github.com/voidlinux/void-packages/pull/10822#discussion_r161549496)...
I would be interested to hear the opinion of @Vaelatern

File: srcpkgs/kitty/template [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Template file for 'kitty'
+pkgname=kitty
+version=0.9.1
+revision=1
+python_version=3
+hostmakedepends="ncurses pkg-config python3-devel wayland-devel wayland-protocols"
+makedepends="gettext-devel glfw-devel harfbuzz-devel libxkbcommon-devel
+python3-devel wayland-devel wayland-protocols"
```
Comment (cr6git) [inline srcpkgs/kitty/template]: you forgot a space before `python3-devel`

File: srcpkgs/kitty/template [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+do_build() {
+	if [ -n "$CROSS_BUILD" ]; then
+		export CFLAGS="-I${XBPS_CROSS_BASE}/usr/include/python3.6m"
+		sed -i "s|-fpie|-fpie ${CFLAGS}|g" setup.py
+	fi
+	case "$XBPS_TARGET_MACHINE" in
+		*-musl)
+			sed -i 's/pow10/pow_10/g' kitty/parser.c;;
+	esac
+	python3 setup.py linux-package --prefix=${DESTDIR}/usr
```
Comment (cr6git) [inline srcpkgs/kitty/template]: it could be interesting to see the build output with `--verbose`

**Comment by julio641742** [comment]
Ready for another review :+1:

**Comment by SolitudeSF** [comment]
`0.10.0` is out

**Comment by cr6git** [comment]
0.10.1 is out

**Comment by julio641742** [comment]
v0.10.1 is ready for another review :+1:

**Pull request merged**